eos-link-host-mali-driver.service

```diff
@@ -0,0 +1,22 @@
+# Symlink the host GL driver for Mali to a place where Flatpak will mount it
+
+[Unit]
+Description=Symlink Mali GL driver for Flatpak
+DefaultDependencies=no
+Conflicts=shutdown.target
+Wants=local-fs.target
+After=local-fs.target
+Before=multi-user.target
+
+# Only run on ARM systems with the Mali driver where we haven't run before
+ConditionArchitecture=arm
```

Comment:
Not that it matters for our current devices, but looking at the other services we have in eos-boot-helper, I suspect we may want to replace this with:
```
ConditionArchitecture=|arm
ConditionArchitecture=|arm64
```

I'm going to merge as is, but @ramcq please confirm whether or not this would be needed on 64-bit ARM.